cloudify-manager-worker/README.md

```diff
@@ -402,7 +334,7 @@ Then you can configure DNS record (ALIAS type), points to this load balancer hos
 ### After values are verified, install the manager worker chart
 
 ```bash
-$ helm install cloudify-manager-worker cloudify-helm/cloudify-manager-worker -f ./cloudify-manager-worker/values.yaml -n NAMESPACE
+$ helm install cloudify-manager-worker cloudify-helm/cloudify-manager-worker --version 0.4.0 -f ./cloudify-manager-worker/values.yaml -n NAMESPACE
```

Comment:
Why specify the version here? is it mandatory in docs to specify the latest? or is it because of the breaking changes?

Reply:
It is not necessary, I just always prefere to locking on specified versions during deploy something.